src/components/panel/panel.js

```diff
@@ -1127,7 +1155,7 @@ MdPanelRef.prototype._configureClickOutsideToClose = function() {
 MdPanelRef.prototype._configureTrapFocus = function() {
   // Focus doesn't remain instead of the panel without this.
   this._panelEl.attr('tabIndex', '-1');
-  if (this._config.trapFocus) {
+  if (this._config['trapFocus']) {
```

Comment:
I wonder if we want to define a `config` object somewhere in this file. It would certainly make it easier to know types and create externs later.